src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -347,72 +352,54 @@ const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
-                    />
-                    {providedExistingCase ? (
-                        <DirectorySelect types={[ElementType.DIRECTORY]} />
-                    ) : (
-                        <UploadNewCase
-                            caseFile={caseFile}
-                            caseFileLoading={caseFileLoading}
-                            handleCaseFileUpload={handleCaseFileUpload}
-                        />
-                    )}
-                    <ImportParametersSection
-                        onChange={handleParamsChange}
-                        currentParameters={currentParameters}
-                        formatWithParameters={formattedCaseParameters}
-                    />
-                    <Grid pt={1}>
-                        {!!apiCallErrorMessage && (
-                            <Alert severity="error">
-                                {apiCallErrorMessage}
-                            </Alert>
-                        )}
-                        {caseFileErrorMessage && (
-                            <Alert severity="error">
-                                {caseFileErrorMessage}
-                            </Alert>
-                        )}
-                    </Grid>
-                </DialogContent>
-                <DialogActions>
-                    <Button onClick={handleCancelStudyCreation}>
-                        <FormattedMessage id="cancel" />
-                    </Button>
-                    <Button
-                        onClick={handleSubmit(handleCreateNewStudy)}
-                        disabled={!isCreationAllowed}
-                        variant="outlined"
-                    >
-                        <FormattedMessage id="validate" />
-                    </Button>
-                </DialogActions>
-            </Dialog>
-        </FormProvider>
+        <CustomMuiDialog
+            titleId={'createNewStudy'}
+            formSchema={schema}
+            formMethods={createStudyFormMethods}
+            removeOptional={true}
+            open={open}
+            onClose={onClose}
+            onSave={handleCreateNewStudy}
+            onCancel={handleCancelStudyCreation}
+            disabledSave={!isCreationAllowed}
+        >
+            <TextInput
+                label={'nameProperty'}
+                name={STUDY_NAME}
+                customAdornment={studyNameAdornment}
+                inputProps={{
+                    autoFocus: true,
+                }}
+                withMargin
+            />
+            <TextInput
+                name={DESCRIPTION}
+                label={'descriptionProperty'}
+                withMargin
+            />
+            {providedExistingCase ? (
+                <DirectorySelect types={[ElementType.DIRECTORY]} />
+            ) : (
+                <UploadNewCase
+                    caseFile={caseFile}
+                    caseFileLoading={caseFileLoading}
+                    handleCaseFileUpload={handleCaseFileUpload}
```

Comment:
Same as below, move all the code related to a case inside this component, including RHF ofc